server.js

```diff
@@ -149,6 +149,34 @@ async function onRequest(req,res) {
 			}
 		}
 
+		// check if the request url is using "auth-worker.js"
+		// when I just looked for "auth-worker.js" it didn't seem to actually find the url
+		if (req.url == "/js/auth-worker.js") {
+			res.writeHead(200, {
+				...HSTSHeader,
+				"Content-Security-Policy":
+			[
+				`default-src ${[
```

Comment:
need to verify if we even need this policy included (if not, let's remove).

Reply:
Is this for the `default-src` or the included `...HSTSHeader`?

Reply:
default-src

Reply:
In this [MDN Doc!](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src) it state that there is no inheritance with `default-src`, and it seems that `default-src 'self'` is the same as a more robust CSP so I am thinking it is not needed/